gluecodium/src/main/resources/templates/jni/utils/JniClassCacheHeader.mustache

```diff
@@ -52,7 +52,7 @@ private:
 };
 
 template <class ... CppTypes>
-class CachedJavaClass : public CachedJavaClassBase
+class JNIEXPORT CachedJavaClass : public CachedJavaClassBase
```

Comment:
class CachedJavaClassBase should be exported, this one - not.

Reply:
Done